elliptic-curve/src/hash2curve.rs

```diff
@@ -0,0 +1,14 @@
+/// Traits for handling hash to curve
+mod group_digest;
+/// Traits for mapping an isogeny to another curve
+/// <https://datatracker.ietf.org/doc/draft-irtf-cfrg-hash-to-curve>
+mod isogeny;
+/// Traits for mapping field elements to points on the curve
+mod map2curve;
+/// Optimized simplified Shallue-van de Woestijne-Ulas methods
+mod osswu;
```

Comment:
Much of the rest of this crate uses toplevel comments in the respective modules for these, rather than commenting them here